frontend/scss/pages/_home_page.scss

```diff
@@ -199,6 +199,48 @@
   margin-top: 50px;
 }
 
+.products-component-container {
+  margin: 0 auto;
+}
+
+.products-component {
+  margin: 20px 5px 20px 5px;
```

Comment:
no big deal but this rule can be `20px 5px;` 20 applies to top/bottom and 5 applies to left/right